src/javascript/db.js

```diff
@@ -302,3 +309,26 @@ export async function getResult(resultId) {
     throw new Error("Error reading document");
   }
 }
+
+export async function getBestScore(sheetId) {
+  try {
+    let snapshot = await resultsCollection
+      .where("uid", "==", store.state.currentUser?.uid)
+      .where("sheetId", "==", sheetId)
```

Comment:
Just to confirm, these `==` actually mean value-equality not the weak JS double-equals-equality right?

Reply:
Should be. See [query operators](https://firebase.google.com/docs/firestore/query-data/queries?authuser=0#query_operators).